EIPS/eip-draft_erc_5219_resolve_mode.md

```diff
@@ -25,7 +25,7 @@ Contracts wishing to use ERC-5219 as their ERC-4804 resolve mode must implement
 /// @dev IDecentralizedApp is the ERC-5219 interface
 interface IERC5219Resolver is IDecentralizedApp {
     // @notice The ERC-4804 resolve mode
-    // @dev    This MUST return "5219" for ERC-5219 resolution (case-insensitive). The other options, as of writing this, are "auto" for automatic resolution, or "manual" for manual resolution.
+    // @dev    This MUST return "5219" (0x00000000000000000000000035323139) for ERC-5219 resolution (case-insensitive). The other options, as of writing this, are "auto" for automatic resolution, or "manual" for manual resolution.
```

Comment:
Thanks for the quick update.  I just test this in remix, and the return in hex should be "0x3532313900000000000000000000000000000000000000000000000000000000" for `bytes32` in Solidity, i.e., the code

```
    function resolveMode3() public returns (bytes32) {
        return "5219";
    }
```
(review is removed to observe the return value in solidity).

Could you please double check?

Reply:
```suggestion
    // @dev    This MUST return "5219" (0x3532313900000000000000000000000000000000000000000000000000000000) for ERC-5219 resolution (case-insensitive). The other options, as of writing this, are "auto" for automatic resolution, or "manual" for manual resolution.
```